stack/indexer/rpm/wazuh-indexer.spec

```diff
@@ -68,12 +68,8 @@ mkdir -p ${RPM_BUILD_ROOT}%{LOG_DIR}
 mkdir -p ${RPM_BUILD_ROOT}%{LIB_DIR}
 mkdir -p ${RPM_BUILD_ROOT}%{SYS_DIR}
 
-# Set up required files
-if [ "%{_base}" = "s3" ];then
-    curl -kOL https://packages-dev.wazuh.com/stack/indexer/base/%{INDEXER_FILE}
-else
-    cp /root/output/%{INDEXER_FILE} ./
-fi
+cp /opt/%{INDEXER_FILE} ./ ;\
```

Comment:
Same here.

Reply:
The base source directory change is due to the fact that the /root directory is a directory mounted at the boot of the container, for this reason, everything that exists in the image in the /root directory is replaced by the contents of the directory mounted.

For this reason, I decided to place the base in another dir and continue the installation with this reference